packages/dd-trace/src/exporters/common/agent-info-exporter.js

```diff
@@ -49,6 +50,7 @@ class AgentInfoExporter {
   }
 
   _export (payload, writer = this._writer, timerKey = '_timer') {
+    incrementCountMetric(TELEMETRY_EVENTS_ENQUEUED_FOR_SERIALIZATION, {}, payload.length)
```

Comment:
Does this belong here since this is a common base?

Reply:
true, good catch